apps/condo/domains/common/components/EmptyListView.tsx

```diff
@@ -4,37 +4,50 @@ import { useRouter } from 'next/router'
 import { useIntl } from '@core/next/intl'
 import { Button } from './Button'
 import { EmptyIcon } from './EmptyIcon'
+import React from 'react'
 
 const EmptyListViewContainer = styled.div`
-  display: flex;
-  width: 100%;
-  height: 100%;
-  justify-content: center;
-  align-items: center;
+    display: flex;
+    width: 100%;
+    height: 100%;
+    justify-content: center;
+    align-items: center;
 `
-export const EmptyListView = () => {
+
+export interface IEmptyListProps {
+    title: string,
+    text: string,
+    createRoute: string,
+    createLabel: string
+}
+
+export const EmptyListView: React.FC<IEmptyListProps> = ({ title, text, createRoute, createLabel }) => {
     const intl = useIntl()
     const router = useRouter()
 
+    const EmptyListHeader = intl.formatMessage({ id: title })
```

Comment:
We use some special suffixes for translations (Message, Label). `Header` not in that list.
And it's better to get a translated word as an argument.

Reply:
Done